guacamole-ext/src/main/resources/org/apache/guacamole/protocols/ssh.json

```diff
@@ -12,6 +12,10 @@
                 {
                     "name"  : "port",
                     "type"  : "NUMERIC"
+                },
+                {
+                    "name"  : "host-key",
+                    "type"  : "MULTILINE"
```

Comment:
Does this value contain multiple lines? I thought each host was defined by a single line.

Reply:
Yeah, I think you're right.

Reply:
Fixed.